### DISCUSSION TITLE
How to train Vocoder for Multi-Speaker?

### DISCUSSION THREAD

**Discussion by kin0303** [coqui-ai/TTS#1961] [General Q&A] [answered]
Hi, I've questions about Vocoder

On https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html#training-a-vocoder-model, just explained to train one speaker

Is it vocoder train using one of the speakers or it can be multispeaker training? If can, how to train them?

**Comment by erogol** [comment, marked as answer]
you just need to train the vocoder with all the audio files you have. There is nothing more you need to do.

**Reply by kin0303** [reply]
> you just need to train the vocoder with all the audio files you have. There is nothing more you need to do.

Is it by combining all speakers or training per speaker?

**Reply by erogol** [reply]
combining all
